.eslintrc.js

```diff
@@ -0,0 +1,190 @@
+module.exports = {
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "ignorePatterns": ["/*", "!/src"],
```

Comment:
Since TSConfig is configured to target the src directory, typescript-eslint must also.  This means we can't use .ts files in the project root or lint files in the project root.

Alternately we could change tsconfig to target project root instead of `src`.  I didn't want to get in to any changes though that might cause a difference in the emitted files.